<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>762</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>demo-cube-payment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditAddress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelConnected">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelAddress">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>9</y>
      <width>55</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOutput">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>531</width>
      <height>641</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabVend">
     <attribute name="title">
      <string>vend</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonTestConnection">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>testconnection</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonReport">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>report (brief)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonGetParameters">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>getparameters</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUpdateconfiguration">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>updateconfiguration</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLoadworkkeys">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>loadworkkeys</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLoadmasterkeys">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>loadmasterkeys</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSettlement">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>settlement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClear">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRunreset">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>runreset</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButtonTransaction">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>transaction</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelTransactionType">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>type</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxTransactionType">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>111</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>purchase</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>purchase-with-cashback</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>refund</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>void</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonReportFull">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>report (full)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancelTransaction">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>cancel transaction</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPapp">
     <attribute name="title">
      <string>papp</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabJpay">
     <attribute name="title">
      <string>jpay</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonJpayLogin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Token</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditJpayToken">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>201</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>insert here token after login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayLogout">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>logout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayChangepassword">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>changepassword</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayUpdateconfiguration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>updateconfiguration</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayLoadmasterkeys">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>loadmasterkeys</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpaySettlement">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>settlement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayLoadworkkeys">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>loadworkkeys</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayTestConnection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>testconnection</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayReport">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>report</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayGetParameters">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>getparameters</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayClear">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushJpayButtonRunreset">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>runreset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayTransaction">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>transaction</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditJpayPassword">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>12345678</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>187</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayResetpassword">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>resetpassword</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayDirectpayment">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>250</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>directpayment</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayKeepalive">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>keepalive</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonJpayDisplay">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>90</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>display</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>680</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxPort">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65536</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>30</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Message to be sent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Cube response</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Message presets</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAutoReconnect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto reconnect</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearOutput">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>680</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <zorder>lineEditAddress</zorder>
   <zorder>labelAddress</zorder>
   <zorder>textEditOutput</zorder>
   <zorder>tabWidget</zorder>
   <zorder>pushButtonSend</zorder>
   <zorder>spinBoxPort</zorder>
   <zorder>labelPort</zorder>
   <zorder>pushButtonConnect</zorder>
   <zorder>textEditInput</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>checkBoxAutoReconnect</zorder>
   <zorder>labelConnected</zorder>
   <zorder>pushButtonClearOutput</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
